<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="data">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QTableView" name="data_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Column information</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="name_edit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="formulaLabel">
               <property name="text">
                <string>Formula</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="formula_edit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="add_column_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Add Column</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="create_plot_button">
             <property name="text">
              <string>Create Plot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionAdd_column"/>
    <addaction name="actionAdd_calculated_column"/>
    <addaction name="actionAdd_row"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_column"/>
    <addaction name="actionDelete_row"/>
   </widget>
   <addaction name="menuTable"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_column">
   <property name="text">
    <string>Add column</string>
   </property>
  </action>
  <action name="actionAdd_calculated_column">
   <property name="text">
    <string>Add calculated column</string>
   </property>
  </action>
  <action name="actionAdd_row">
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="actionDelete_column">
   <property name="text">
    <string>Delete column</string>
   </property>
  </action>
  <action name="actionDelete_row">
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
